src/js/containers/agencyV2/accountSpending/TableContainer.jsx

```diff
@@ -112,14 +114,24 @@ const TableContainer = (props) => {
         if (currentPage !== 1) {
             changeCurrentPage(1);
         }
-        else {
-            fetchSpendingByCategoryCallback();
+        else if (currentPage === 1) {
+            const hasParamChanged = (
+                prevSort !== sort ||
+                prevOrder !== order ||
+                prevPage !== currentPage ||
+                prevPageSize !== pageSize
```

Comment:
I think we also need to check if `props.type` has changed - it's still not making a new request when I switch tabs (e.g. from Budget Functions to Program Activities)